### PR TITLE
Onboarding Banking App

### PR DESCRIPTION
**Pull request by lisambamberg**

### CODE REVIEW

File: OnboardingExperience/Program.cs [hunk 1]
```diff
@@ -6,9 +6,86 @@ class Program
...
+            Console.WriteLine("Hello! Welcome to the TrueCoders Onboard bank experince!");
+
+            user.IsAccountHolder = AskBoolQuestion("Are you an account holder?");
+
+            user.FirstName = AskQuestion("What is your first name?");
+            Console.WriteLine($"Hello {user.FirstName}.");
+
+            user.LastName = AskQuestion("What is your last name?");
+            Console.WriteLine($"Welcome {user.FirstName} {user.LastName}.");
+
```
Comment (codytwinton) [inline OnboardingExperience/Program.cs]: Unnecessary whitespace. Only needs one line of separation.

File: OnboardingExperience/Program.cs [hunk 2]
```diff
@@ -6,9 +6,86 @@ class Program
...
+                {
+                    Console.WriteLine("That's great! We'll need to gather information about you to get started!");
+                    return true;
+                }
+
+                else if ("no" == answer)
+                {
+                    Console.WriteLine("That is unfortunate.");
+                    Console.ReadLine();
+                    Environment.Exit(-1);
```
Comment (codytwinton) [inline OnboardingExperience/Program.cs]: Move this logic back up into your `Main` method.

File: OnboardingExperience/Program.cs [hunk 3]
```diff
@@ -6,9 +6,86 @@ class Program
...
+
+                Console.WriteLine("You must answer yes or no");
+            }
+        }
+
+        static int AskPinQuestion(string question)
+        {
+            while (true)
+            {
+                int length = 3;
```
Comment (codytwinton) [inline OnboardingExperience/Program.cs]: Consider making this a parameter, so this could be modified by the caller.

File: OnboardingExperience/Program.cs [hunk 4]
```diff
@@ -6,9 +6,86 @@ class Program
...
+                var response = AskQuestion(question);
+                char[] ResponseArray = response.ToCharArray();
```
Comment (codytwinton) [inline OnboardingExperience/Program.cs]: Use the keyword `var` here.

File: OnboardingExperience/Program.cs [hunk 5]
```diff
@@ -6,9 +6,86 @@ class Program
...
+                bool PinBool = Int32.TryParse(response, out var num);
```
Comment (codytwinton) [inline OnboardingExperience/Program.cs]: Use the keyword `var` here.

File: OnboardingExperience/Program.cs [hunk 6]
```diff
@@ -6,9 +6,86 @@ class Program
...
+
+                if (!PinBool)
+                {
+                    Console.WriteLine($"Sorry that is not valid. " +
+                            $"Please try again. Please enter your 3 digit personal Pin number.");
+                    Console.ReadLine();
```
Comment (codytwinton) [inline OnboardingExperience/Program.cs]: I'd remove this `ReadLine()`

File: OnboardingExperience/Program.cs [hunk 7]
```diff
@@ -6,9 +6,86 @@ class Program
...
+                    continue;
+                }
+                return num;
+
+            }
+
+
+        }
+
+
```
Comment (codytwinton) [inline OnboardingExperience/Program.cs]: Unnecessary whitespace.

File: OnboardingExperience/User.cs [hunk 8]
```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
```
Comment (codytwinton) [inline OnboardingExperience/User.cs]: Remove unused `using` directives.

**Comment by lisambamberg** [comment]
I have reviewed your comments and have made changes. Thank you.
